ui/components/app/edit-gas-display/edit-gas-display.component.js

```diff
@@ -168,7 +171,7 @@ export default function EditGasDisplay({
             {t('gasDisplayAcknowledgeDappButtonText')}
           </Button>
         )}
-        {showRadioButtons && (
+        {networkAndAccountSupport1559 && showRadioButtons && (
```

Comment:
This seems to be redundant because showRadioButtons includes the check for networkAndAccountSupport1559

Reply:
Good shout!